banzai/tic-tac-toe.rb

```diff
@@ -0,0 +1,108 @@
+# this gem injects a color method to the string when it is displayed by console
+require 'colorize' 
```

Comment:
Add instructions for requirements in Readme.md

Reply:
Si, la clase sobre gemas la veremos recién hoy